01week/datatypes.js

```diff
@@ -0,0 +1,43 @@
+//this is my method to display the current day and time
+function displayDayAndTime() {
+  const x = new Date();
+  return x;
+}
+
+console.log(displayDayAndTime());
+
+//this is my method to convert a number to a string
+function numToString(num){
+  return num.toString();
+}
+console.log(typeof numToString(5));
+
+//this is my method to convert a string to a number
+function strToNumber(str){
+  return Number(str);
+}
+console.log(typeof strToNumber('hello'));
+
+//this is my method to take in different datatypes and prints out the types
+console.log(typeof numToString('typeofmethod'));
+
+//this is my method to add two numbers together
+function sumOfTwoNumber(num1, num2){
+  return num1 + num2
+}
+
+//this is my method to run only when two things are true
+
```

Comment:
I need this function to evaluate to true, not test if something is equal to true based on being larger or smaller than a number. Evaluating to true and being equal to true are different. look up evaluating to true.